src/scenes/Jarjestajat/Jarjestaja/Hakemukset/Muutospyynto/components/MuutospyyntoWizardPerustelut.js

```diff
@@ -19,10 +19,14 @@ import {
 import { getRemovalFormStructure } from "../../../../../../services/lomakkeet/perustelut/tutkinnot";
 import FormSection from "../../../../../../components/03-templates/FormSection";
 import PerustelutOpiskelijavuodet from "./Perustelut/PerustelutOpiskelijavuodet";
+import YhteenvetoLiitteet from "./Yhteenveto/YhteenvetoLiitteet";
+import PerustelutLiitteet from "./Perustelut/PerustelutLiitteet";
+import wizard from "../../../../../../i18n/definitions/wizard";
 
 const MuutospyyntoWizardPerustelut = ({
   changeObjects,
   intl,
+  intl: { formatMessage },
```

Comment:
There shouldn't be any need for this line. `intl` is already defined and it should contain the _formatMessage_ method.

Reply:
Some reason does not work without -> need to studied when all left localizations are done.